### ISSUE TITLE
Copy the list object to clipboard 

### ISSUE THREAD

**Issue by aazz0099a** [spyder-ide/spyder#11768]
### What steps will reproduce the problem?

1. Extract one column value from Dataframe
2. Column: type = list, size = 1, value = [scatter]

It is a accidentially human error.
I am not suppose to copy the [scatter] when using the plotly to plot that column object.
But just fyi, in case it is potential incident =)

### Traceback
```python-traceback
  File "C:\Users\Steven\tmp\lib\site-packages\spyder\plugins\variableexplorer\widgets\collectionseditor.py", line 1037, in copy
    obj = self.delegate.get_value(idx)
  File "C:\Users\Steven\tmp\lib\site-packages\spyder\plugins\variableexplorer\widgets\collectionseditor.py", line 1386, in get_value
    return self.parent().get_value(name)
  File "C:\Users\Steven\tmp\lib\site-packages\spyder\plugins\variableexplorer\widgets\collectionseditor.py", line 1439, in get_value
    value = self.shellwidget.get_value(name)
  File "C:\Users\Steven\tmp\lib\site-packages\spyder\plugins\ipythonconsole\widgets\namespacebrowser.py", line 99, in get_value
    timeout=CALL_KERNEL_TIMEOUT).get_value(name)
  File "C:\Users\Steven\tmp\lib\site-packages\spyder_kernels\comms\commbase.py", line 542, in __call__
    call_dict, call_data, self._comm_id)
  File "C:\Users\Steven\tmp\lib\site-packages\spyder\plugins\ipythonconsole\comms\kernelcomm.py", line 114, in _get_call_return_value
    call_dict, call_data, comm_id)
  File "C:\Users\Steven\tmp\lib\site-packages\spyder_kernels\comms\commbase.py", line 420, in _get_call_return_value
    return self._sync_error(reply['value'])
  File "C:\Users\Steven\tmp\lib\site-packages\spyder_kernels\comms\commbase.py", line 480, in _sync_error
    error_wrapper.raise_error()
AttributeError: 'tuple' object has no attribute 'raise_error'
```

## Versions

* Spyder version: 4.0.1 
* Python version: 3.7.3
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 7

### Dependencies

```
atomicwrites >=1.2.0         :  1.3.0 (OK)
chardet >=2.0.0              :  3.0.4 (OK)
cloudpickle >=0.5.0          :  0.8.0 (OK)
diff_match_patch >=20181111  :  20181111 (OK)
intervaltree                 :  None (OK)
IPython >=4.0                :  7.4.0 (OK)
jedi =0.14.1                 :  0.14.1 (OK)
nbconvert >=4.0              :  5.4.1 (OK)
numpydoc >=0.6.0             :  0.8.0 (OK)
pexpect >=4.4.0              :  4.8.0 (OK)
pickleshare >=0.4            :  0.7.5 (OK)
psutil >=0.3                 :  5.6.1 (OK)
pygments >=2.0               :  2.3.1 (OK)
pylint >=0.25                :  2.3.1 (OK)
pyls >=0.31.2;<0.32.0        :  0.31.7 (OK)
zmq >=17                     :  18.0.0 (OK)
qdarkstyle >=2.7             :  2.8 (OK)
qtawesome >=0.5.7            :  0.5.7 (OK)
qtconsole >=4.6.0            :  4.6.0 (OK)
qtpy >=1.5.0                 :  1.7.0 (OK)
rtree >=0.8.3                :  0.8.3 (OK)
sphinx >=0.6.6               :  1.8.5 (OK)
spyder_kernels >=1.8.1;<2.0.0:  1.8.1 (OK)
watchdog                     :  None (OK)
cython >=0.21                :  0.29.6 (OK)
matplotlib >=2.0.0           :  3.0.3 (OK)
numpy >=1.7                  :  1.16.2 (OK)
pandas >=0.13.1              :  0.24.2 (OK)
scipy >=0.17.0               :  1.2.1 (OK)
sympy >=0.7.3                :  1.3 (OK)
```

**Comment by aazz0099a** [comment]
my code attached fyi =)
hsbc_10 is copied to clipboard.
``````````````````````````````````````````````````
import talib as ta
import yfinance as yf

hsbc = yf.Ticker('HSBC')
df_hsbc = pd.DataFrame(hsbc.history(period ='max', interval ='1d', start='2019-01-01'))
hsbc_close = np.array(df_hsbc['Close'])
hsbc_close.round(0)

sma_10 = ta.SMA(hsbc_close, timeperiod = 10)
sma_20 = ta.SMA(hsbc_close, timeperiod = 20)

hsbc_10 = [go.Scatter(x= df_hsbc.T.columns,y= sma_10.tolist())]
hsbc_20 = [go.Scatter(x= df_hsbc.T.columns,y= sma_20.tolist())]

**Comment by ccordoba12** [comment]
Thanks for reporting. This will be fixed in our next version (4.0.1) to be released in a couple of days.
